src/account/AccountInfo.js

```diff
@@ -263,4 +263,41 @@ export default class AccountInfo {
             this._toProtobuf()
         ).finish();
     }
+
+    /**
+     * @returns {string}
+     */
+    toString() {
+        return JSON.stringify(this.toJSON());
+    }
+
+    /**
+     * @param {string} accountInfo
+     * @returns {AccountInfo}
+     */
+    fromString(accountInfo){
+        return new AccountInfo(JSON.parse(accountInfo));
+    }
+
+    /**
+     * @returns {{}}
+     */
+    toJSON() {
```

Comment:
Can you add a test for this.

Reply:
added a test. discovered that fromString doesn't work.